### ISSUE TITLE
Remove Web Password Option

### ISSUE THREAD

**Issue by cjlee89** [pi-hole/docker-pi-hole#1229]
I know `WEBPASSWORD` got implemented and I can remove the password all together by editing the setupVars file after docker creation. When updating the container, the `WEBPASSWORD` is put back. Is there a way we could get a `LEAVE_THE_FRONT_DOOR_OPEN` (or similar) variable to allow for permanent removal of the web password even after updates?

**Comment by PromoFaux** [comment]
Set `WEBPASSWORD` to be blank, i.e:

```yml
environment:      
      WEBPASSWORD:
      PIHOLE_DNS_:8.8.8.8;8.8.4.4
      #other variables
```

**Comment by PromoFaux** [comment]
Oh actually, hold that thought... it might not work

**Comment by cjlee89** [comment]
> Oh actually, hold that thought... it might not work

I was actually about to try it when I realized I have in the past. When updating, it then just creates a random password for me.

**Comment by PromoFaux** [comment]
Sure thing - I'll take a look later when I am at home

**Comment by cjlee89** [comment]
That did not allow me to login at all and I didn't see any admin password generated in the logs.

Running `pihole -a -p` without a password removed it until I restarted.

**Comment by rdwebdesign** [comment]
I tested:

I was able to reproduce this behavior using just `WEBPASSWORD:   ` (nothing after the colon).
It resulted in a password being created. This is the same as not using the environment variable at all.

I **wasn't** able to reproduce the behavior using `WEBPASSWORD: ''` (empty string).
This second attempt resulted in a passwordless access (the same as using `pihole -a -p` and setting an empty password).
